server/controllers/api/apiController.ts

```diff
@@ -57,7 +57,7 @@ export default class ApiController {
       res.status(201).json({
         ...report,
         urn: `uk:gov:hmpps:pre-sentence-service:report:${report.id}`,
-        url: `${config.domain}/api/v1/report/${report.id}`,
+        url: `${config.domain}/short-format/${report.id}`,
```

Comment:
shouldn't this be reportType here? this can be `record-of-oral` too?
```suggestion
        url: `${config.domain}/${reportType}/${report.id}`,
```